mozilla_vpn_privacy_notice/en-US.md

```diff
@@ -0,0 +1,42 @@
+## <span class="privacy-header-firefox">Mozilla VPN</span> <span class="privacy-header-policy">Privacy Notice</span>
+
+Effective July 15, 2020
+{: datetime="2020-07-15" }
+
+## At Mozilla, we design products with your privacy in mind.
```

Comment:
Let's change the heading style for this one to match with the rest.

Reply:
@litkaemily for this one - each word start with a capital letter so it matches with the other heading style: `At Mozilla, We Design Products with Your Privacy in Mind.` unless it is purposely done this way because the heading is a sentence.

Reply:
I just updated the case throughout, so it should be consistent now